src/main/java/Ui.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+/**
+ * Ui class handles the bot's response to the user's input.
+ */
+public class Ui {
+    public Ui() {
+
+    }
```

Comment:
Maybe it could better if the Scanner was initialised in the Ui class? So that you don't have to create a new scanner every time you read a command. You can just create one in your Ui class and call on that!